hooks/pre-commit

```diff
@@ -1,4 +1,14 @@
 #!/bin/bash
 set -e
+if [[ $TRAVIS_PYTHON_VERSION ]]; then
+    VERSION=$TRAVIS_PYTHON_VERSION
+else
+    VERSION=$(python3 -c 'import sys; v = sys.version_info; print("{}.{}".format(v[0], v[1]));')
+fi
 python3 -m flake8 -j auto
+if [ $VERSION = '3.4' ]; then
```

Comment:
@miaekim we use py34 mainly. but py33 is not include some standard library such as `enum`, `pathlib`. so we install polyfill from pypi. polyfill is not std lib, so import-order can not decide correct order.